cmd/mayactl/app/command/volume_delete.go

```diff
@@ -34,45 +33,29 @@ var (
 	`
 )
 
-// CmdVolumeDeleteOptions stores the input parameters
-type CmdVolumeDeleteOptions struct {
-	volName string
-}
-
 // NewCmdVolumeDelete creates a new OpenEBS Volume
 func NewCmdVolumeDelete() *cobra.Command {
-	options := CmdVolumeDeleteOptions{}
-
 	cmd := &cobra.Command{
 		Use:   "delete",
 		Short: "Deletes a Volume",
 		Long:  volumeDeleteCommandHelpText,
 		Run: func(cmd *cobra.Command, args []string) {
-			util.CheckErr(options.ValidateVolumeDelete(cmd), util.Fatal)
+			util.CheckErr(options.Validate(cmd), util.Fatal)
 			util.CheckErr(options.RunVolumeDelete(cmd), util.Fatal)
 		},
 	}
 
 	cmd.Flags().StringVarP(&options.volName, "volname", "", options.volName,
 		"unique volume name.")
 	cmd.MarkPersistentFlagRequired("volname")
-
 	return cmd
 }
 
-//ValidateVolumeDelete validates the arguments passed
-func (c *CmdVolumeDeleteOptions) ValidateVolumeDelete(cmd *cobra.Command) error {
-	if c.volName == "" {
-		return errors.New("--volname is missing. Please specify an unique name")
-	}
-	return nil
-}
-
 //RunVolumeDelete will initiate deletion of volume from maya-apiserver
```

Comment:
will initiate the process of deleting a volume from...

Reply:
done.